qcodes/dataset/sqlite/queries.py

```diff
@@ -1764,7 +1766,7 @@ def _both_zero(run_id: int,
 
         log.info(f'Succesfully updated run number {run_id}.')
 
-    actions: Dict[Tuple[bool, bool], Callable]
+    actions: Dict[Tuple[bool, bool], Callable[..., Any]]
```

Comment:
there are the arguments `run_id, conn, guid_comps`
```suggestion
    actions: Dict[Tuple[bool, bool], Callable[[int, ConnectionPlus, Dict[str, Any]], None]]
```